frontend/app/components/wp-table/wp-table.directive.html

```diff
@@ -152,7 +152,7 @@
               lang="::{{column.custom_field && column.custom_field.name_locale || locale}}"
               wp-edit-field="::getTableColumnName(row.object, column.name)"
               wp-edit-field-wrapper-classes="'-small -shrink'"
-              display-classes="::[row.level > 0 && column.name == 'subject' && 'icon-context icon-arrow-right5 icon-small']"
+              display-classes="[row.level > 0 && column.name == 'subject' && 'icon-context icon-arrow-right5 icon-small']"
               columns="::columns"
               field-index="::$index"
```

Comment:
@oliverguenther can you please assess whether this one time binding (`field-index="::$index"`) might cause trouble e.g. when a column is added/removed?